linux_os/guide/services/ntp/var_multiple_time_pools.var

```diff
@@ -18,3 +18,4 @@ options:
     amazon: "0.rhel.pool.ntp.org,1.rhel.pool.ntp.org,2.rhel.pool.ntp.org,3.rhel.pool.ntp.org"
     ubuntu: "0.ubuntu.pool.ntp.org,1.ubuntu.pool.ntp.org,2.ubuntu.pool.ntp.org,3.ubuntu.pool.ntp.org"
     debian: "0.debian.pool.ntp.org,1.debian.pool.ntp.org,2.debian.pool.ntp.org,3.debian.pool.ntp.org"
+    nist: "time.nist.gov"
```

Comment:
The name of the var is "var_multiple_time_pools.var" this is only one that seems a bit misleading. I wondering if we want to skip adding it here.